javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
Here's a query that could serve as a migration script:
`UPDATE jv_snapshot s SET s.version = (SELECT COUNT(*) + 1 FROM (SELECT * FROM jv_snapshot) s2 WHERE s.global_id_fk = s2.global_id_fk and s2.snapshot_pk < s.snapshot_pk)`
It is SQL-vendor agnostic and accomplishes the task. I'm not that sure about performance, but that could be described in the release notes.